src/components/AppHeader/AppHeader.css

```diff
@@ -2,6 +2,8 @@
 
 .appBar {
   background-color: var(--backgroundAltColor);
+  height: auto;
+  padding: 8px 2.4rem;
```

Comment:
@imorente the reasons for using `px` are consistency with existing CSS and the fact that `rem`s and `em`s can cause half-pixel boundaries, which are undesirable. I have mixed feelings - those are both significant issues, but the accessibility problem is a hard one to ignore. For now, I think we should stick with `px` for the simple reason that we shouldn't make a transition to relative units ad-hoc, even if we do end up using them at some point in the future.